packages/internal/src/build/babel/web.ts

```diff
@@ -11,12 +11,16 @@ import {
   getCommonPlugins,
   registerBabel,
   RegisterHookOptions,
+  parseConfigFiles,
+  getPathsFromConfig,
 } from './common'
 
 export const getWebSideBabelPlugins = (
   { forJest, forVite }: Flags = { forJest: false, forVite: false }
 ) => {
   const rwjsPaths = getPaths()
+  //get the config object from the file
+  const config = parseConfigFiles()
```

Comment:
`config` isn't too descriptive; can we change it to something like `tsConfigs`? (Plural because this is both right?)

```suggestion
  const tsConfigs = parseConfigFiles()
```

Reply:
same as the other two comments, this can be regular JS, `jsconfig.json` file

Reply:
See https://github.com/redwoodjs/redwood/pull/7575#discussion_r1196940890